src/block_disagg/block_disagg_read.c

```diff
@@ -214,6 +202,16 @@ __block_disagg_read_multiple(WT_SESSION_IMPL *session, WT_BLOCK_DISAGG *block_di
                     block_meta->delta_count = get_args.delta_count;
                     block_meta->checksum = checksum;
```

Comment:
This should be the latest checksum so it should be assigned before we change checksum variable to the previous_checksum?